src/router/index.ts

```diff
@@ -207,6 +208,40 @@ const router = createRouter({
 			component: TaskDetailView,
 			props: route => ({ taskId: Number(route.params.id as string) }),
 		},
+		{
+			path: '/tasks/create',
+			name: 'task.create',
+			beforeEnter: async (to, from, next) => {
+				const taskStore = useTaskStore()
+				const authStore = useAuthStore()
+				// Need to get settings populated first!
+				await authStore.checkAuth()
+
+				const defaultProjectId = authStore.settings.defaultProjectId
+				if(!defaultProjectId) {
+					return false
+				}
+
+				const title = to.query.title || 'Share'
+				const description = to.query.description as string
+				
+				const task = await taskStore.createNewTask({
+					title,
+					projectId: defaultProjectId,
+				})
+		
+				if (description) {
+					task.description = description
+					await taskStore.update(task)
```

Comment:
Please create the task with the description directly, no need to update it after creating it.

Reply:
Is there an interface for that? I could not figure it out just now.